src/schema/objects/columns.yaml

```diff
@@ -42,13 +42,29 @@ age:
     for privacy purposes.
   type: number
   unit: year
+age_category:
+  name: age_category
+  display_name: Age category
+  description: |
+    For non-numeric description of the age (e.g. developmental stage) a string can be provided.
+    We RECOMMEND to use the [UBERON life cycle stage](http://purl.obolibrary.org/obo/UBERON_0000105) to specify it.
+  type: string
+age_type:
+  name: age_type
+  display_name: Age type
+  description: |
+    Type of age measurement (e.g. `postnatal`, `fetal`, `gestational`). If not specified, `postnatal` is assumed.
+  type: string
+  enum:
+    - $ref: objects.enums.postnatal.value
+    - $ref: objects.enums.fetal.value
+    - $ref: objects.enums.gestational.value
```

Comment:
Thank you @ree-gupta !  I wonder if we better propose this as a PR against current bids version, just keep `surgery_date` for the BEP032 but propose `age_category` and `age_type` . 

It also would complement
- https://github.com/bids-standard/bids-specification/pull/1791
which I am yet to finish.

Then it would give it a better chance for discussion etc.

Also I guess they should be added as recommended to
https://github.com/bids-standard/bids-specification/blob/master/src/schema/rules/tabular_data/modality_agnostic.yaml#L2

Reply:
Done! Also, `birthdate` is another column suggested by the BEP032.